src/Tokenizer.js

```diff
@@ -112,6 +112,8 @@ export function delimit (type) {
 	while (character = scan())
 		if (character === type)
 			break
+		else if (character === 92)
```

Comment:
I pushed a commit that fixes the other delimiter issues, and includes this.

Reply:
Ok, I've adjusted this PR - it only contains new tests now 👍